auto_gptq/quantization/__init__.py

```diff
@@ -1,2 +1,3 @@
+from .config import *
```

Comment:
Can you import here specifically the classes/functions/vars you'd like to expose? I think it is best to avoid star imports.